packages/cadl-vs/src/VSExtension.cs

```diff
@@ -195,12 +195,12 @@ private static string GetDevelopmentCadlServerPath()
             var args = new string[] { "--stdio" };
 #if DEBUG
             // Use local build of cadl-server in development (lauched from F5 in VS)
-            if (InDevelopmentMode())
-            {
-                var options = Environment.GetEnvironmentVariable("CADL_SERVER_NODE_OPTIONS");
-                var module = GetDevelopmentCadlServerPath();
-                return ("node.exe", new string[] { module, options }.Concat(args).ToArray(), env);
-            }
+            //if (InDevelopmentMode())
+            //{
+            //    var options = Environment.GetEnvironmentVariable("CADL_SERVER_NODE_OPTIONS");
+            //    var module = GetDevelopmentCadlServerPath();
+            //    return ("node.exe", new string[] { module, options }.Concat(args).ToArray(), env);
+            //}
```

Comment:
Is this meant to be commented out now or was this just for testing?

Reply:
testing :) I'll revert